detectors/detector_template/app.py

```diff
@@ -24,18 +22,26 @@ def predict():
     predictions = []
     for filename in video_list:
         score = 0.5
-        video = filename.rsplit('/',1)[-1]
```

Comment:
there's potential for `video` to be an UnboundLocalVariable without giving it a default value and referring to it outside the try/except

Reply:
this should be fixed